config/expressive.config.php

```diff
@@ -0,0 +1,57 @@
+<?php
+return array(
+    'dependencies' => array (
```

Comment:
I think you can update this file. [currently](https://github.com/RWOverdijk/AssetManager/blob/master/config/module.config.php#L5) we use the class names for all of these.